Code/Framework/AzFramework/AzFramework/Input/Buses/Requests/InputTextEntryRequestBus.h

```diff
@@ -30,6 +30,13 @@ namespace AzFramework
         {
             AZStd::string m_initialText;   //!< The virtual keyboard's initial text
             AZStd::string m_titleText;     //!< The virtual keyboard's title text
+#if defined(CARBONATED)
+            bool m_showSendOnReturnKey = false; //!< Whether to show the send button on the return key
```

Comment:
having a comment that matches the bool name doesnt really help dev understanding - If you could lets change the comment to be something like // Whether we set the native keyboard return key type to Send

Reply:
Changed comment